addons/auth_signup/views/auth_signup_login_templates.xml

```diff
@@ -3,8 +3,8 @@
         <template id="auth_signup.login" inherit_id="web.login" name="Sign up - Reset Password">
             <xpath expr="//button[@type='submit']" position="after">
                 <div class="justify-content-between mt-2 d-flex small">
-                    <a t-if="signup_enabled" t-attf-href="/web/signup?{{ keep_query() }}">Don't have an account?</a>
-                    <a t-if="reset_password_enabled" t-attf-href="/web/reset_password?{{ keep_query() }}">Reset Password</a>
+                    <a t-if="signup_enabled" t-attf-href="/web/signup?{{ keep_query() if keep_query else '' }}">Don't have an account?</a>
```

Comment:
is this linked to #87491 ?

Reply:
Yes, but I won't commit this part. I just added it so I could correctly launch the server while developing